test/performance/latency/LatencyTestTypes.hpp

```diff
@@ -129,6 +129,27 @@ class LatencyDataType : public eprosima::fastdds::dds::TopicDataType
         return false;
     }
 
+    void set_size(
```

Comment:
This method has been declared, but I see no implementation.

Reply:
This method has not been included finally and instead the combination of delete/create is being used to resize the type. Consequently, this declaration has been erased in 759afcbf9cb2c567b30b3b3475617648754d5895.